src/name/bizna/ocarmsim/SRAMRegion.java

```diff
@@ -80,4 +79,19 @@ public void maybeWriteNVRAM() throws IOException {
 		}
 	}
 
+	public byte[] getSramArray() {
+		return sramArray;
+	}
+
+	public void setSramArray(byte[] sramArray) {
+		this.sramArray = sramArray;
+	}
+
+	public byte[] getNvramArray() {
+		return nvramArray;
+	}
+
+	public void setNvramArray(byte[] nvramArray) {
+		this.nvramArray = nvramArray;
+	}
```

Comment:
Allowing the arrays to change at runtime makes the region size no longer constant. `PhysicalMemorySpace` assumes that regions don't change in size.